### PR TITLE
[incrementals-publisher] Add Incrementals.jenkins.io

### PR DESCRIPTION
**Pull request by halkeye**
#### Is this a new chart
- [x] `CODEOWNERS` is updated
- [x] If secrets are needed, `.sops.yaml` must be updated with appropriated public gpg key ID, and full public gpg key must be provided to @olblak

#### Which issue this PR fixes

  - fixes # [INFRA-XXXX](https://issues.jenkins-ci.org/projects/INFRA/issues/INFRA-XXXX)

#### Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [X] Chart Version bumped
- [x] Variables are documented in the README.md
- [X] Title of the PR starts with chart name (e.g. `[stable/mychartname]`)

### CODE REVIEW

**Comment by halkeye** [comment]
i'm super lazy, basic readme generated by helm-docs

**Review by markyjackson-taulia — APPROVED** [review]
I will wait for another review and merge in my am (unless merged by someone else)

File: charts/incrementals-publisher/templates/deployment.yaml
Comment (halkeye) [reply charts/incrementals-publisher/templates/deployment.yaml:50]: I don't know what a webhook token is.

It's the exact same code as before just wrapped in it's own container

It was talking to jenkins before

Comment (timja) [reply charts/incrementals-publisher/templates/deployment.yaml:50]: Azure was protecting it with a pre shared token (a feature of function webhooks)

Code wasn’t needed before

Comment (halkeye) [reply charts/incrementals-publisher/templates/deployment.yaml:50]: https://github.com/jenkins-infra/community-functions/blob/06074904372cd979ea6ca5b959102635a352b654/incrementals-publisher/index.js#L147

It's always had it.

I can absolutely add the preshared key if I know more about it. Like how it was provided (query string?)

Comment (timja) [reply charts/incrementals-publisher/templates/deployment.yaml:50]: That’s how the function authenticates you ci.jenkins.io not how the function authenticates ci.jenkins.io.

It was in a query string I think you can see it in the pipeline library code, I would change it to a Authorization: Token header so it doesn’t show up in logs.

Comment (halkeye) [reply charts/incrementals-publisher/templates/deployment.yaml:50]: Ooooo

okay yea I can add that pretty easy. I do it tomorrow

Comment (halkeye) [reply charts/incrementals-publisher/templates/deployment.yaml:50]: https://github.com/jenkins-infra/charts-secrets/pull/7

Comment (halkeye) [reply charts/incrementals-publisher/templates/deployment.yaml:50]: https://github.com/jenkins-infra/incrementals-publisher/pull/1

**Review by timja — APPROVED** [review]

**Review by markyjackson-taulia — APPROVED** [review]

**Comment by halkeye** [comment]
YOLO!

merging

**Pull request merged**
